streampark-console/streampark-console-service/src/test/java/org/apache/streampark/common/util/RegexTest.java

```diff
@@ -28,6 +26,8 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 
 class RegexTest {
```

Comment:
BTW, `RegexTest#classLoader` and `RegexTest#flinkVersion` seem quite tricky and fail locally. How can we verify these tests correctly?

Reply:
This is really tricky to solve, and the local environment needs to meet the requirements. We can delete this testcase first.

Reply:
Sounds reasonable. But I don't have time to investigate more. Perhaps open as a follow-up.